css/style.css

```diff
@@ -1,19 +1,157 @@
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+body, footer {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
+
+/*Navigation*/
+.navbar{
+  background-color: white;
+  position: sticky;
+}
+.navbar .logo {
+  height: 60px;
+  margin-left: 30px;
+  float: left;
+}
+.navbar a{
+  float: left;
+  margin-top: 0;
+  display: block;
+  color: grey;
+  text-align: center;
+  padding: 14px 20px;
+  background-color: white;
+}
+.navbar a:hover{
+  color: orange;
+}
+.right {
+  float: right;
+}
+a{
+  text-decoration: none;
+}
+.navbar a.current-tab{
+  color: black;
+}
+.navbar .icon {
+  display: none;
+}
+
+
+/*Introduction General styles*/
+.intro{
+  background-image: url(/img/first-background.jpg);
+  background-size: cover;
+	background-repeat: no-repeat;
+	background-position: center bottom;
+  width: 100%;
+  height: auto;
+  overflow: auto;
+}
+  
+.intro .content{
+  color: white;
+  text-align: center;
+  padding: 100px;
+  padding-top: 0px;
+}
+.intro h1{
+  font-size: 50px;
+  margin-top: 200px;
+  font-weight: 100;
+}
+.intro h3{
+  font-size: 20px;
+  margin-top: 20px;
+  font-weight: 100;
+}
+
+/*Button - general styles*/
+.button{
+  background-color: orange;
+  color: white;
+}
+.button:hover{
+  color: red;
+}
+
+/*Main text of the landing page - General styles*/
+.main{
+  display: flex;
+  justify-content: center;
+  margin-left: 20px;
+  margin-right: 20px;
+  vertical-align: middle;
+  flex-direction: row;
+  
+}
+/*Icons styles*/
+.main .devices{
+  width: 100%;
+  height: auto;
+  padding: 115px;
+  margin-top: 0px;
+}
+.main .coffee {
+  width: 100%;
+  height: auto;
+  padding:115px;
+  margin-top: 0px;
+}
+.main .refill{
+  width: 100%;
+  height: auto;
+  padding:115px;
+  margin-top: 0px;
 
-body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+}
+/*General styles*/
+.statement{
+  text-align: center;
+  margin-top: 100px;
+}
+.statement-for-icons{
+  text-align: center;
+  padding-right: 0px;
 }
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+/*Line break*/
+hr.solid {
+  border-top: 3px solid grey;
+  margin-left: 20px;
+  margin-right: 20px;
+  margin-top: 0px;
+}
 
+/*Footer*/
+.footer{
+  display: flex;
+  justify-content: center;
+}
 
+/*Changing screen size* - General style changes as a result*/
+@media screen and (max-width: 1200px) {
+  .navbar a:not(:nth-child(0)) {display: none;}
+  .navbar a.icon {
+    float: right;
+    display: block;
+  }
+  .main{
+    display: flex;
+    flex-direction: column;
+    margin-left: 0px;
+    margin-right: 250px;
```

Comment:
This margin-right is likely to cause you trouble. Try and keep everything balanced if you want elements to be centered and re-position themselves in a reliable way when you resize the screen.
Giving appropriate sizes to your icons and using Flexbox on the container should help you do this nicely.

Reply:
CSS updated to use flexbox